net.sapples.LiveCaptions.json

```
@@ -0,0 +1,108 @@
+{
+    "app-id" : "net.sapples.LiveCaptions",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "livecaptions",
+    "finish-args" : [
+        "--device=dri",
```

```suggestion
        "--device=dri",
        "--share=ipc",
```

it's required for x11.